gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 
-gradlePlugin-kotlin = "1.8.10"
-gradlePlugin-android = "4.0.1"
+gradlePlugin-kotlin = "1.8.20"
+gradlePlugin-android = "4.1.3"
```

Comment:
If I remember correctly, KGP 1.8.20 now requires AGP to be <= 4.1.3. Could not build the project without bumping the version here.